### PR TITLE
FIXES #1565 [Menu Driven Program]

### LINKED ISSUE #1565: Circular-Doubly-Linked-List
I would like to create a dynamic menu driven program for Circular Doubly Linked List in C++ for GSSoC'2020. Please assign this task to me.

### PR DESCRIPTION
**Pull request by HardevKhandhar**
Fixes #1565

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Menu Driven Program
- 

#### Languages Used:
- C++
- 

#### Files Added:
- Circular-Doubly-Linked-List.cpp
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by asha15** [comment]
please provide the issue number

**Comment by HardevKhandhar** [comment]
#1565

**Comment by jainaman224** [comment]
There are multiple files for the same program. Please self review the PR once. To get your PR quickly.

**Comment by HardevKhandhar** [comment]
> There are multiple files for the same program. Please self review the PR once. To get your PR quickly.

@jainaman224 
I am sorry for the mistake, it had 2 files. 

I have deleted one file that was not required. Now it contains only one file.

Thank you

File: Circular_Doubly_Linked_List/Circular_Doubly_Linked_List.cpp [hunk 1]
```diff
@@ -1,131 +1,290 @@
...
+            return 0;
+            break;
+
+        default:
+            cout << "Invalid Choice!" << endl;
+            break;
+        }
+    }
+    return 0;
+}
```
Comment (jainaman224) [inline Circular_Doubly_Linked_List/Circular_Doubly_Linked_List.cpp]: Add sample input and output.
Add a single new line at the of file.

File: Circular_Doubly_Linked_List/Circular_Doubly_Linked_List.cpp [hunk 2]
```diff
@@ -1,131 +1,290 @@
...
+        cout << "4. Delete node at front" << endl;
+        cout << "5. Delete node at last" << endl;
+        cout << "6. Delete node at given position" << endl;
+        cout << "7. Display elements of linked list" << endl;
+        cout << "8. Exit" << endl;
+        cout << "\nEnter your choice: ";
+        cin >> choice;
+        switch(choice)
+        {
+        case 1:
```
Comment (jainaman224) [inline Circular_Doubly_Linked_List/Circular_Doubly_Linked_List.cpp]: Incorrect indentation.

File: Circular_Doubly_Linked_List/Circular_Doubly_Linked_List.cpp [hunk 3]
```diff
@@ -1,131 +1,290 @@
...
+    else{
+        while(trav -> rptr != HEAD)
+        {
+            previous = trav;
+            trav = trav -> rptr;
+        }
+        cout << "Element " << trav -> data << " is removed." << endl;
+        previous = trav -> lptr;
+        previous -> rptr = HEAD;
+        }
```
Comment (jainaman224) [inline Circular_Doubly_Linked_List/Circular_Doubly_Linked_List.cpp]: Incorrect indentation.

File: Circular_Doubly_Linked_List/Circular_Doubly_Linked_List.cpp [hunk 4]
```diff
@@ -1,131 +1,290 @@
...
-        t=t->next;
-        cnt++;
-    }
-    n->prev=temp;
-    n->next=temp->next;
-    temp->next=n;
-    t->prev=n;
+        HEAD = NULL;
+    }
+    else{
```
Comment (jainaman224) [inline Circular_Doubly_Linked_List/Circular_Doubly_Linked_List.cpp]: Add space before `{`.

**Comment by HardevKhandhar** [comment]
Dear @jainaman224 Sir

Checklist is updated
PR Template is updated
Changes are committed according to the instructions.
Indentations checked & sample input/output added.

You may now review the PR again.

Thank you

**Pull request closed without merging**